angular.json

```diff
@@ -51,7 +51,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```

Comment:
@skos001 This property causes the build to fail and according to the docs it has been deprecated since Angular 11, defaulting to `true` now, so I _think_ it's OK to leave it out here...

Reply:
I think so... ng build was succeeding locally before, and is still succeeding. Looks good 😄